Assets/MixedRealityToolkit/_Core/Devices/MixedReality/WindowsMixedRealityController.cs

```diff
@@ -53,7 +53,7 @@ public WindowsMixedRealityController(ControllerState controllerState, Handedness
         public IMixedRealityInputSource InputSource { get; private set; }
 
         /// <inheritdoc/>
-        public Dictionary<DeviceInputType, InteractionMapping> Interactions { get; private set; }
+        public Dictionary<DeviceInputType, IInteractionMapping> Interactions { get; private set; }
```

Comment:
This just struck me... it is very possible that a given controller may have multiple controls (ex: Game Controllers generally have two thumbsticks) for a given DeviceInputType… is this going to bite us?

Reply:
DeviceInputType will GROW,  So I fully expect to add Thumbstick 1 / Thumbstick 2 when we do Xbox controllers.
@StephenHodgson does disagree with me on this, but I cannot think of another way to uniquely identify differert hardware types in a fasterway than this lookup.  Most SDK's (Xbox Gamepad is a good example) use an enum, not an ID, then apply different "GetValue" methods for each control.

Reply:
I agree, I think we should be using something different as the key

Reply:
OK, then let's finish up this PR which was to address a different issue. Then create a task to revisit this later, as it is currently working.

Reply:
> DeviceInputType will GROW, So I fully expect to add Thumbstick 1 / Thumbstick 2 when we do Xbox controllers.

We should most def not do this.

Reply:
> We should most def not do this.

How do you see it being implemented?